mcs/class/referencesource/System/compmod/system/diagnostics/DiagnosticsConfiguration.cs

```diff
@@ -168,8 +168,10 @@ internal static class DiagnosticsConfiguration {
         }
         
         private static SystemDiagnosticsSection GetConfigSection() {
-            SystemDiagnosticsSection configSection = (SystemDiagnosticsSection) PrivilegedConfigurationManager.GetSection("system.diagnostics");
-            return configSection;
+            object o = PrivilegedConfigurationManager.GetSection("system.diagnostics");
```

Comment:
Yeah, that would have been better. They wouldn't accept this code upstream anyway, as they wanted to correct it differently, so I don't think it is a big deal.